incubator/openvpn/templates/NOTES.txt

```diff
@@ -0,0 +1,20 @@
+OpenVPN is now starting.
+
+Please be aware that certificate generation is variable and may take some time (minutes).
+check pod status via:
+POD_NAME=`kubectl get pods -l type=openvpn | awk END'{ print $1 }'` \
+&& kubectl log $POD_NAME --follow
+
+LoadBalaner ingress creation can take some time as well.
+check service status via: kubectl get svckubectl get svc
+
+Once the external IP is available and all the server certificates are generated create client key .ovpn files by pasting the following into a shell:
+
+POD_NAME=`kubectl get pods -l type=openvpn | awk END'{ print $1 }'` \
+&& SERVICE_NAME=`kubectl get svc -l type=openvpn | awk END'{ print $1 }'` \
+&& SERVICE_IP=$(kubectl get svc --namespace {{ .Release.Namespace }} $SERVICE_NAME -o jsonpath='{.status.loadBalancer.ingress[0].ip}') \
+&& KEY_NAME=kubeVPN \
+&& kubectl exec -it $POD_NAME /etc/openvpn/setup/newClientCert.sh $KEY_NAME $SERVICE_IP \
+&& kubectl exec -it $POD_NAME cat /usr/share/easy-rsa/pki/$KEY_NAME.ovpn > $KEY_NAME.ovpn
```

Comment:
Not all of the kubectl commands in here are passing the namespace so this doesnt work in custom namespaces ATM.